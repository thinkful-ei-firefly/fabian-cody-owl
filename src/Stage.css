.stage{
  display: flex;
  background-color: lightgray;
  flex-grow: 3;
}

.stage-participant{
  background-color: lightgray;
  width: 50%;
  text-align: center;
}