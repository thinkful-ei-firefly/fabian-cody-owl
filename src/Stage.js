import React from 'react';
import './Stage.css';

class Stage extends React.Component{
  render(){
    return (
      <div className="stage-participant">
        <h2>{this.props.name}</h2>
        <img src={this.props.avatar} alt="avatar"/>
      </div>
    )
  }
}

export default Stage;