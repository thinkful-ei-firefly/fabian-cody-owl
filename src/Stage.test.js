import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage';

describe('<Stage /> ', () =>{
  it('Renders without Crashing', ()=> {
    const div = document.createElement('div');
    ReactDOM.render(<Stage name="Hohn Smith" avatar="www.avatar.com"/>, div)
    ReactDOM.unmountComponentAtNode(div);
  })

  it('Renders <Stage /> as expected', ()=> {
    const tree = renderer.create(<Stage name="Hohn Smith" avatar="www.avatar.com"/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})