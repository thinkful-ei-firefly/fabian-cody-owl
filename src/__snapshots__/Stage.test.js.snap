// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage />  Renders <Stage /> as expected 1`] = `
<div
  className="stage-participant"
>
  <h2>
    Hohn Smith
  </h2>
  <img
    alt="avatar"
    src="www.avatar.com"
  />
</div>
`;
